### PR TITLE
appliance_console: dialog for static ipv6 settings

### PR DESCRIPTION
**Pull request by isimluk**
## Dependencies
- [x] depends on ~~https://github.com/ManageIQ/linux_admin/pull/176~~ or https://github.com/ManageIQ/linux_admin/pull/181
- [x] depends on https://github.com/ManageIQ/linux_admin/pull/177
- [x] depends on https://github.com/ManageIQ/linux_admin/pull/178

## This is how it looks:
```
IPv6: Static Network Configuration

Enter the new static network configuration settings.

Enter the IP Address: |2620:52:0:2804:3fec:900f:ca31:a115| 
Enter the IPv6 prefix length: |64| 
Enter the Default gateway: |fe80:52:0:2804::1fc| 
Enter the Primary DNS: |10.38.5.26| fe80:52:0:2804::1fd
Enter the Secondary DNS (Enter 'none' for no value): none
Enter the Domain search order: |brq.redhat.com redhat.com|
```
and then
```
Static Network Configuration

        IP Address:      2620:52:0:2804:3fec:900f:ca31:a115/64
        Gateway:         fe80:52:0:2804::1fc
        Primary DNS:     fe80:52:0:2804::1fd
        Secondary DNS:   
        Search Order:    brq.redhat.com redhat.com

Apply static network configuration? (Y/N)
```

### CODE REVIEW

File: lib/gems/pending/appliance_console.rb [hunk 1]
```diff
@@ -230,6 +231,57 @@ module ApplianceConsole
...
+              say("\nNetwork interface failed to start using the values supplied.")
+              press_any_key
+              next
+            end
+
+            say("\nAfter completing the appliance configuration, please restart #{I18n.t("product.name")} server processes.")
+            press_any_key
+          end
+
+
```
Comment (jrafanie) [inline lib/gems/pending/appliance_console.rb]: extra line here

**Comment by jrafanie** [comment]
LGTM, once the dependencies are merged and we bump the linux_admin gem is updated, we can merge this.  Marking as WIP for now.

File: lib/gems/pending/appliance_console.rb (hunk 1 above)
Comment (isimluk) [inline lib/gems/pending/appliance_console.rb]: Woo, nice catch. Removed.

**Comment by isimluk** [comment]
> LGTM, once the dependencies are merged and we bump the linux_admin gem is updated,
> we can merge this. 

So, here I see two options

(1) I beg someone to release linux_admin gem, update the dependencies, merge this. And then I continue working on DHCPv6, possibly requiring another rebase of linux_admin.
(2) We just merge this. As we know an release is approaching and we will bump & release anyway.

Former seems more work, with two releases. Latter seems less work, with single release.

What do you think? What are the other options? Do we want to create the rule to always release linux_admin gem, once we want to put a new method in there?

**Comment by carbonin** [comment]
>(1) I beg someone to release linux_admin gem, update the dependencies, merge this. And then I continue working on DHCPv6, possibly requiring another rebase of linux_admin.

What work is still pending for DHCPv6? I think it would make more sense to get all the IPv6 related changes into one linux_admin release if possible. If you think there is a lot more work to be done, we could try for multiple.

>(2) We just merge this. As we know an release is approaching and we will bump & release anyway.

I think I would rather have all the dependencies sorted out before merging so testing can start when the PR is merged. Plus I think it makes more sense to add the version bump here in the same PR as the changes that depend on it.

Also, are you still seeing issues with gem conflicts @isimluk ? When we release linux_admin we will need to change the gemspec here and then we would start seeing those issues on this PR.

**Comment by isimluk** [comment]
In general, I would love to work in smaller increments. As it allows me to deliver greater quality (can test early with nightly appliances, can do extra stuff - like move "IP stuff to get into a separate appliance console"). However, I can adapt and wait. No problem. :)

**Comment by isimluk** [comment]
Updated linux_admin dependency, this is now ready to be merged!

Thanks!

**Comment by simaishi** [comment]
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1212896

**Pull request merged**